hbase-backup/src/test/java/org/apache/hadoop/hbase/backup/TestIncrementalBackup.java

```diff
@@ -101,11 +102,14 @@ public TestIncrementalBackup(Boolean b) {
   public void ensurePreviousBackupTestsAreCleanedUp() throws Exception {
     TEST_UTIL.flush(table1);
     TEST_UTIL.flush(table2);
-    TEST_UTIL.flush(table1_restore);
 
     TEST_UTIL.truncateTable(table1).close();
     TEST_UTIL.truncateTable(table2).close();
-    TEST_UTIL.truncateTable(table1_restore).close();
+
+    if (TEST_UTIL.getAdmin().tableExists(table1_restore)) {
```

Comment:
sometimes if our restore fails, there's no table1_restore, and we fail to do the cleanup here. this can make it tricky to track down the actual test failure b/c it will show up as the primary reason for the test failure. decided to only clean this table up if it already exists so it's easier to track down the test-specific failure